src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/ConversionCompletionProvider.cs

```diff
@@ -35,8 +35,8 @@ public ConversionCompletionProvider()
 
         protected override ImmutableArray<CompletionItem> GetCompletionItemsForTypeSymbol(ITypeSymbol container, bool isAccessedByConditionalAccess, SemanticModel semanticModel, int position)
         {
-            var containerIsNullable = container.IsNullable() || // var int? id = null; id.$$                
-                (container.IsValueType && isAccessedByConditionalAccess); // var p = new System.Diagnostics.Process(); p?.Id.$$;
+            var containerIsNullable = container.IsNullable() || // int? id; id.$$ -> (byte?)
+                (container.IsValueType && isAccessedByConditionalAccess); // System.Diagnostics.Process p; p?.Id.$$ -> (byte?)
```

Comment:
fwiw, commenting like this is super annoying to read.  i would just break these into simple statements with comments above htem explaining what they're doing.